app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,57 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      include Response
+      include ExceptionHandler
+
+      def show
+        json_response(find_volunteer)
+      end
+
+      def create
+        json_response(Volunteer.create!(volunteer_params))
```

Comment:
create a variable and before you render it as json. Try not to render the create function. It works but it erases consistency and readability